src/antares/service/local_services/thermal_local.py

```diff
@@ -31,7 +39,74 @@ def update_thermal_properties(
         raise NotImplementedError
 
     def get_thermal_matrix(self, thermal_cluster: ThermalCluster, ts_name: ThermalClusterMatrixName) -> pd.DataFrame:
-        raise NotImplementedError
+        if ts_name.value == "series":
+            time_serie_type = TimeSeriesFileType.THERMAL_SERIES
+        elif ts_name.value == "modulation":
+            time_serie_type = TimeSeriesFileType.THERMAL_MODULATION
+        elif ts_name.value == "data":
+            time_serie_type = TimeSeriesFileType.THERMAL_DATA
+        elif ts_name.value == "CO2Cost":
+            time_serie_type = TimeSeriesFileType.THERMAL_CO2
+        elif ts_name.value == "fuelCost":
+            time_serie_type = TimeSeriesFileType.THERMAL_FUEL
+
+        return read_timeseries(
+            time_serie_type,
+            self.config.study_path,
+            area_id=thermal_cluster.area_id,
+            cluster_id=thermal_cluster.properties.group.value,
+        )
 
     def read_thermal_clusters(self, area_id: str) -> List[ThermalCluster]:
-        raise NotImplementedError
+        thermal_dict = IniFile(self.config.study_path, IniFileTypes.THERMAL_LIST_INI, area_name=area_id).ini_dict
+        thermal_clusters = []
+        if thermal_dict:
+            for thermal_cluster in thermal_dict:
+                thermal_properties = ThermalClusterPropertiesLocal(
```

Comment:
This is really really ugly. We should introduce aliases inside the `ThermalClusterProperitesLocal` so inside the `DefaultThermalProperties` class to use pydantic to validate this. And this goes with adding the aliases camel case inside the `API` class.
But this is too complicated for this PR so you can just add a `TODO` saying that we should refactor this

Reply:
Ok, for the last commit